frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/premium.embedding.unit.spec.tsx

```diff
@@ -87,28 +119,14 @@ describe("[EE, with token] embedding settings", () => {
         );
 
         expect(
-          screen.getByLabelText("Cross-Origin Resource Sharing (CORS)"),
+          screen.getByLabelText("Enable Embedding SDK for React"),
+        ).not.toBeChecked();
+        expect(
+          screen.getByLabelText("Enable Embedding SDK for React"),
         ).toBeEnabled();
-      });
-    });
-
-    describe("interactive embedding", () => {
-      it("should not allow going to interactive settings page", async () => {
         expect(
-          within(
-            screen.getByRole("article", {
-              name: "Interactive embedding",
-            }),
-          ).getByRole("button", { name: "Configure" }),
+          screen.getByLabelText("Cross-Origin Resource Sharing (CORS)"),
```

Comment:
This assertion is changed. It used to be asserted as `toBeEnabled` now this input respects `enable-embedding-sdk`, and asserted as `toBeDisabled`